tutorials/ai-core-aiapi-postman-resource/ai-core-aiapi-postman-resource.md

```diff
@@ -119,6 +119,7 @@ Follow the below steps for creating a path prefix through AWS CLI:
 
 [ACCORDION-BEGIN [Step 4: ](Upload dataset to AWS S3 Object Store)]
 
+> **DISCLAIMER:** The following data is a sample data that is being curated only for this tutorial. It is not be circulated or used for any commercial production projects or for experimentation other than the tutorial for SAP AI Core.
```

Comment:
You may want to check and rephrase this part. I think that you mean "it has been created" instead of "is being curated".
Also it should read: "The following data is sample data...", and "It is not to be circulated...".